src/printer.ts

```diff
@@ -1295,12 +1295,27 @@ export class PugPrinter {
 
 		if (this.previousToken?.type === 'dot') {
 			const lastTagToken: TagToken | undefined = previousTagToken(this.tokens, this.currentIndex);
+			let typeAttributeToken: AttributeToken | undefined;
+			if (lastTagToken != null) {
+				const tokens: Token[] = this.tokens.slice(this.tokens.indexOf(lastTagToken) + 1, this.currentIndex - 1);
+				typeAttributeToken = tokens.find((token) => token.type === 'attribute' && token.name === 'type') as
```

Comment:
@ST-DDT Do you know some optimizations for this part?
_Maybe we can have a call about this in the next few days_ :slightly_smiling_face: